### PR TITLE
FBR-286: Integrate hard policy checklists

### PR DESCRIPTION
**Pull request by fiter-julius-oketayot**
## Description
Jira Task: https://fiterio.atlassian.net/browse/FBR-286

### CODE REVIEW

File: fineract-provider/src/main/resources/db/changelog/tenant/commands/edit_hardpolicy_checklist_configuration_tables.sql
Comment (leonardohildt) [inline fineract-provider/src/main/resources/db/changelog/tenant/commands/edit_hardpolicy_checklist_configuration_tables.sql:1]: I think we should move the table creation sentences into a regular migration, and then a changeset can execute the command file to insert or update values in these tables. Is there any reason to do this here?

Comment (fiter-julius-oketayot) [reply fineract-provider/src/main/resources/db/changelog/tenant/commands/edit_hardpolicy_checklist_configuration_tables.sql:1]: I noticed that the table was created already using a similar approach here
https://github.com/fiterlatam/fineract-template/blob/fiter/fb/dev/fineract-provider/src/main/resources/db/changelog/tenant/commands/create_hardpolicy_checklist_configuration_tables.sql

Comment (fiter-julius-oketayot) [reply fineract-provider/src/main/resources/db/changelog/tenant/commands/edit_hardpolicy_checklist_configuration_tables.sql:1]: @leonardohildt I  think that creating the table using the standard liquibase xml command would be a better approach
let me make the update

Comment (leonardohildt) [reply fineract-provider/src/main/resources/db/changelog/tenant/commands/edit_hardpolicy_checklist_configuration_tables.sql:1]: Yeah, probably the other sql was missed in the PR review. We need to keep an eye on how we document the upgrades in the database vs executing a script to insert or update records.

**Review by leonardohildt — APPROVED** [review]

**Pull request merged**